### PR TITLE
[tests-only] [full-ci] Remove tests related to issue-1328,1310,2249 from expected to fail

### PR DESCRIPTION
**Pull request by SwikritiT**
## Related Issue

- https://github.com/owncloud/ocis/issues/1328
- https://github.com/owncloud/QA/issues/753

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>Core-API-Tests-ocis-storage-5</strong>](https://drone.owncloud.com/owncloud/ocis/13545/46/1) failed. Further test are cancelled...

**Comment by phil-davis** [comment]
PR title is "Remove tests related to from expected to fail" - related to what?
Please update the PR title.

**Comment by SwikritiT** [comment]
> PR title is "Remove tests related to from expected to fail" - related to what? Please update the PR title.

Done. Thanks for pointing it out!

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 1]
```diff
@@ -52,11 +49,6 @@ Other free text and markdown formatting can be used elsewhere in the document if
 - [webUIFilesCopy/copyPrivateLinks.feature:20](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesCopy/copyPrivateLinks.feature#L20)
 - [webUIFilesCopy/copyPrivateLinks.feature:21](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesCopy/copyPrivateLinks.feature#L21)
 
-### [Versions Cleanup not available](https://github.com/owncloud/ocis/issues/1310)
-- [webUIFilesActionMenu/versions.feature:37](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesActionMenu/versions.feature#L37)
-- [webUIFilesActionMenu/versions.feature:48](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesActionMenu/versions.feature#L48)
-- [webUIFilesActionMenu/versions.feature:63](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesActionMenu/versions.feature#L63)
```
Comment (SwikritiT) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:58]: These tests were skipped by PR https://github.com/owncloud/web/pull/7282

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 2]
```diff
@@ -31,9 +31,6 @@ Other free text and markdown formatting can be used elsewhere in the document if
 - [webUIFavorites/unfavoriteFile.feature:102](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFavorites/unfavoriteFile.feature#L102)
 - [webUIResharing1/reshareUsers.feature:194](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIResharing1/reshareUsers.feature#L194)
 
-### [file_path property is not unique for a share created with same resource name i.e already present in sharee](https://github.com/owncloud/ocis/issues/2249)
```
Comment (SwikritiT) [inline tests/acceptance/expected-failures-webUI-on-OCIS-storage.md:34]: This test was skipped by PR https://github.com/owncloud/web/pull/7281

**Pull request merged**